pkg/client/src/app/pages/applications/analysis-wizard/set-mode.tsx

```diff
@@ -66,8 +66,9 @@ export const SetMode: React.FunctionComponent<ISetMode> = ({
       </TextContent>
       <FormGroup label={t("wizard.label.analysisSource")} fieldId="sourceType">
         <SimpleSelect
+          toggleAriaLabel="analysis-mode-toggle"
           variant={SelectVariant.single}
-          aria-label="Select user perspective"
+          aria-label="analysis-mode-select"
```

Comment:
FYI, `aria-label` should be a human-readable sentence string, not an identifier. It's what gets read out by screen readers for blind users. "Select analysis mode" or something is actually best here.

For identifiers like this for QE, you can use the `toggleId` prop (SimpleSelect extends most props of PF's Select, and `toggleId` will put an id on the `<button>` used to toggle the select menu).

Reply:
Thanks Mike - Updated to use toggleID for QE & fixed the aria-label.